L/libjulia/build_tarballs.jl

```diff
@@ -31,9 +31,12 @@ function Pkg.Types.is_stdlib(uuid::Base.UUID, julia_version::VersionNumber)
     return false
 end
 
-jllversion=v"1.10.1"
-build_julia(ARGS, v"1.6.3"; jllversion)
-build_julia(ARGS, v"1.7.0"; jllversion)
-build_julia(ARGS, v"1.8.2"; jllversion)
-build_julia(ARGS, v"1.9.0-rc3"; jllversion)
+# WARNING: the current version was built with julia_compat=1.9
+# increase the version when lowering the compat again to build for older julia versions
+# (julia_compat is set in common.jl)
```

Comment:
Will this work, though? Isn't Yggdrasil itself still using Julia 1.7, so will it even be able to use the libjulia_jll updates produced by this PR?

I guess the goal is to avoid rebuilding older versions Julia of the JLL? Of course in general it is good if we don't get new artifacts, though in this particular case no use installs `libjulia_jll`, only Yggdrasil needs it, so perhaps we can get away with just rebuilding it all and that's it...

(I guess being able to reuse artifacts of older builds of a Julia still is in on the wishlist, @giordano ?)